README.md

```diff
@@ -71,25 +72,42 @@ Let's look at a quick example.
 ```js
 Handlebars.registerPartial('namePartial', document.getElementById("partial-template").innerHTML)
 function renderMain() {
-  var template = Handlebars.compile(document.getElementById("main-template").innerHTML);
-  var html = template({name: 'Gordon Ramsay'});
+  var template = document.getElementById("main-template").innerHTML;
+  var templateFunction = Handlebars.compile(template);
+  var html = templateFunction({name: 'Gordon Ramsay'});
 }
 ```
 
 By default, the partial will receive the same context object as the template that calls it, so our partial has the same access to `name` as the main template does.
 
+## Sneak Peek
+
+You'll be building out 3 screens: New Recipe, Show Recipe, Edit Recipe. When you're done, a user should be able to:
+1. Visit the page and see a form to create a new recipe, fill it out and submit it (New Recipe)
+2. See the recipe details and a link to edit the recipe (Show Recipe)
+3. Click the link to Edit the recipe and see the form rendered with pre-filled values (Edit Recipe)
+3. Update the values in the edit form and submit it to change the recipe (Edit Recipe)
+4. View the updated recipe details (Show Recipe)
```

Comment:
You're doing two enumerations of 3 points (the 4th can probably be cut). It makes me think that a table might present this data with less room for confusion.

Reply:
@sgharms Hey Steven! Sorry for the delay. What are you thinking would be the column headers of the table? One reason I used the numbers was so that the desired user flow would be clear.